code-challenges.test.js

```diff
@@ -57,3 +142,10 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
+//Pseudocode
+// Make function called firstVowelIndex
+//input: "learn", "academy", "challenges"
+//output: 1, 0, 2
+//Built in method that is specific to strings to identify first vowel in word and return that values index to us.
+
+const firstVowelIndex = () => {}
```

Comment:
I need to see a valid attempt on all problems.  There is very little pseudocode and only the skeleton of a function here.  Please update this problem showing your process, what you tried, how you tried it and what results you are getting back from your tests.